theme/brb/views/lib/footer.hbs

```diff
@@ -17,6 +17,10 @@
 					<a href="/datenschutz" target="_blank" aria-label="{{$t "global.text.dataProtection"}} Schul-Cloud Brandenburg PDF {{$t "global.link.openInNewTabWarning"}}" rel="noopener">{{$t "global.text.dataProtection"}}</a>
 					<span> - </span>
 				</li>
+				<li>
+					<a href="mailto:{{getConfig "SC_CONTACT_EMAIL"}}?subject=Schul-Cloud%20Brandenburg%20Anfrage" target="_blank" aria-label="{{$t "global.link.contact"}} {{$t "global.link.openInNewTabWarning"}}" rel="noopener">{{$t "global.link.contact"}}</a>
```

Comment:
Like in nuxt (or better server, as nuxt takes the ENV values from server), please make sure here that on the legacy client the correct values for SC_CONTACT_EMAIL are given in the release ticket